.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+.github/**/*    @Ed-Fi-Alliance-OSS/ods-tools-automation-owners
+build.ps1   @Ed-Fi-Alliance-OSS/ods-tools-automation-owners
```

Comment:
As a reminder: when this change goes into `main`, then anyone can edit the `.github` files or `build.ps1`, and anyone can review: but before a merge can happen, then one of Suganya, Jason, or Stephen needs to approve the merge. Jason and Stephen usually shouldn't be the first person to review in such cases; for example if Suganya is on vacation, then it would be good for someone else on the team to do a first review, and then @jasonh-edfi or @stephenfuqua  do a final review. In that case, it is fine for Jason or Stephen to complete the merge, because another developer on the team has already signed off on it as well.